<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendAndReceive</class>
 <widget class="QWidget" name="SendAndReceive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color:#14274E;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1025</width>
        <height>634</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#394867</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;微软雅黑&quot;;
color:#F1F6F9;</string>
         </property>
         <property name="title">
          <string>串口设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="4" column="1">
           <widget class="QComboBox" name="cb_stopBit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
color:white;
}</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color: white;
}</string>
            </property>
            <property name="text">
             <string>串口</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cb_dataBit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
color:white;
}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color: white;
}</string>
            </property>
            <property name="text">
             <string>波特率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cb_checkBit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
color:white;
}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_portBaud">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
color:white;
}</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color: white;
}</string>
            </property>
            <property name="text">
             <string>校验位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color: white;
}</string>
            </property>
            <property name="text">
             <string>数据位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_portName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
color:white;
}</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color: white;
}</string>
            </property>
            <property name="text">
             <string>停止位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="btn_openSport">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="btn_flash">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;微软雅黑&quot;;
color:#F1F6F9;</string>
          </property>
          <property name="title">
           <string>接收框</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextEdit" name="te_rec">
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;微软雅黑&quot;;
color:#F1F6F9;</string>
          </property>
          <property name="title">
           <string>发送框</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QComboBox" name="cb_historySend">
             <property name="styleSheet">
              <string notr="true">background-color: #9BA4B4;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_send">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#9BA4B4;
</string>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTextEdit" name="te_send"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;微软雅黑&quot;;
color:#F1F6F9;</string>
         </property>
         <property name="title">
          <string>接收设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>清空消息</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cb_autoWrap">
            <property name="text">
             <string>自动换行</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rbtn_rec_HEX">
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbtn_rec_ASCII">
            <property name="text">
             <string>ASCII</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cb_showTime">
            <property name="text">
             <string>显示时间</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_followRoll">
            <property name="text">
             <string>跳至底部</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;微软雅黑&quot;;
color:#F1F6F9;</string>
         </property>
         <property name="title">
          <string>发送设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="3" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbtn_send_HEX">
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="cb_sendTimed">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QSpinBox" name="sb_timeSet"/>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbtn_send_ASCII">
            <property name="text">
             <string>ASCII</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>发送编码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_encode"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
